dlt/pipeline/pipeline.py

```diff
@@ -458,7 +459,7 @@ def run(
 
         # normalize and load pending data
         if self.list_extracted_resources():
-            self.normalize(loader_file_format=loader_file_format)
+            self.normalize(loader_file_format=loader_file_format, schema_evolution_settings=schema_evolution_settings)
```

Comment:
don't pass it to normalizer. just update schema here

Reply:
I did this more or less, I do pass it here, but updating it in that function